### PR TITLE
[BLOG] Add versioned-hdf5-performance

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-versioned-hdf5-performance-quansight.vercel.app/blog/versioned-hdf5-performance). Link to [blog post on old page.](https://labs.quansight.org/blog/2020/09/versioned-hdf5-performance).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by MarsBarLee** [comment]
To fix this build error ([link to Vercel build error](https://vercel.com/quansight/labs/C8K62MKyd9J9YsKeWkxo8a3Cc8ip#:~:text=23%3A54%3A13.520-,error%20%5BError%3A%20%5Bnext%2Dmdx%2Dremote%5D%20error%20compiling%20MDX%3A,that%20can%20start%20a%20name%2C%20such%20as%20a%20letter%2C%20%60%24%60%2C%20or%20%60_%60,-23%3A54%3A13.521))
```
error [Error: [next-mdx-remote] error compiling MDX:
Unexpected character `1` (U+0031) before name, expected a character that can start a name, such as a letter, `$`, or `_`
<10x file writing speed.
^
```
Change `<` to `{'<'}`, a JSX string.

File: apps/labs/posts/versioned-hdf5-performance.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+title: 'Performance of the Versioned HDF5 Library'
+published: September 11, 2020
+author: melissa-mendonca
+description: "In this post, we'll show some of the performance analysis we did while developing the library, hopefully making the case that reading and writing versioned HDF5 files can be done with a nice, intuitive API while being as efficient as possible. The tests presented here show that using the Versioned HDF5 library results in reduced disk space usage, and further reductions in this area can be achieved with the use of HDF5/h5py-provided compression algorithms."
+category: [PyData Ecosystem]
```
Comment (noatamir) [inline apps/labs/posts/versioned-hdf5-performance.md:6]:
```suggestion
category: [PyData ecosystem]
```

File: apps/labs/posts/versioned-hdf5-performance.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+---
+title: 'Performance of the Versioned HDF5 Library'
+published: September 11, 2020
+author: melissa-mendonca
+description: "In this post, we'll show some of the performance analysis we did while developing the library, hopefully making the case that reading and writing versioned HDF5 files can be done with a nice, intuitive API while being as efficient as possible. The tests presented here show that using the Versioned HDF5 library results in reduced disk space usage, and further reductions in this area can be achieved with the use of HDF5/h5py-provided compression algorithms."
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/versioned-hdf5-performance/feature.png
```
Comment (noatamir) [inline apps/labs/posts/versioned-hdf5-performance.md:8]: @MarsBarLee this image is really low quality and looks pixelated. Can you please generae a similar feature image that is sharper?

**Comment by MarsBarLee** [comment]
Ok, added a new, higher quality feature image.

File: apps/labs/posts/versioned-hdf5-performance.md (hunk 2 above)
Comment (pavithraes) [reply apps/labs/posts/versioned-hdf5-performance.md:8]: Done :)

File: apps/labs/posts/versioned-hdf5-performance.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+For now, you can check out the
+[documentation](https://deshaw.github.io/versioned-hdf5/) for more details on
+what is supported and how the library is built.
+
+The Versioned HDF5 library was created by the [D. E. Shaw
+group](https://www.deshaw.com/) in conjunction with
+[Quansight](https://www.quansight.com/).
+
+<img src="/posts/versioned-hdf5-performance/black_logo_417x125.png" width="200" class="center"/>
```
Comment (pavithraes) [inline apps/labs/posts/versioned-hdf5-performance.md:133]: This did not get centered in the preview, maybe we can try:

```suggestion
<div align="center">
  <img src="/posts/versioned-hdf5-performance/black_logo_417x125.png" width="200" />
</div>
```

**Review by pavithraes — APPROVED** [review]
Thanks, @MarsBarLee! This is looking good to merge, so I'll approve!

I've left one suggestion that need not block this PR, so feel free to self-merge after considering the suggestion. :)

File: apps/labs/posts/versioned-hdf5-performance.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+In a [previous post](https://labs.quansight.org/blog/2020/08/introducing-versioned-hdf5/), we introduced the Versioned HDF5 library, which implements a mechanism for storing binary data sets in a versioned way that feels natural to users of other version control systems, and described some of its features. In this post, we'll show some of the performance analysis we did while developing the library, hopefully making the case that reading and writing versioned HDF5 files can be done with a nice, intuitive API while being as efficient as possible. The tests presented here show that using the Versioned HDF5 library results in reduced disk space usage, and further reductions in this area can be achieved with the use of HDF5/[h5py](https://www.h5py.org)-provided compression algorithms. That only comes at a cost of {'<'}10x file writing speed.
```
Comment (pavithraes) [inline apps/labs/posts/versioned-hdf5-performance.md:17]:
```suggestion
In a [previous post](https://labs.quansight.org/blog/introducing-versioned-hdf5/), we introduced the Versioned HDF5 library, which implements a mechanism for storing binary data sets in a versioned way that feels natural to users of other version control systems, and described some of its features. In this post, we'll show some of the performance analysis we did while developing the library, hopefully making the case that reading and writing versioned HDF5 files can be done with a nice, intuitive API while being as efficient as possible. The tests presented here show that using the Versioned HDF5 library results in reduced disk space usage, and further reductions in this area can be achieved with the use of HDF5/[h5py](https://www.h5py.org)-provided compression algorithms. That only comes at a cost of {'<'}10x file writing speed.
```

**Review by pavithraes — CHANGES_REQUESTED** [review]
Oh, just noticed an important change.

**Comment by MarsBarLee** [comment]
@pavithraes Thanks for catching that! I'll go ahead with a squash merge.

**Pull request merged**